addon/templates/components/frost-radio-button.hbs

```diff
@@ -0,0 +1,9 @@
+{{#radio-button
+    radioId=radioId
+    radioClass=radioClass
+    value=value
+    groupValue=groupValue
```

Comment:
Can we handle this via a group wrapper with a yield block?  e.g.

```
{{radio-button-group id=x class=y}}
  {{#radio-button value=z}}
    {{label}}
  {{/radio-button}}
{{radio-button-group}}
```

I think this gives us more interface options down the line.